app/views/macros/showExamples.njk

```diff
@@ -17,7 +17,7 @@
     <div class="govuk-width-container">
       <div class="govuk-heading-m">
         <h3 id="heading-{{ exampleSlug }}" class="app-!-di">{{ heading | safe }}</h3>
-        <a href="{{ path }}" class="govuk-link govuk-!-margin-left-1 govuk-!-font-size-16">
+        <a href="{{ path }}{{ legacy_query }}" class="govuk-link govuk-!-margin-left-1 govuk-!-font-size-16">
```

Comment:
These don't appear to be working – I think because the macro doesn't have access to local variables in the same way, only variables that are passed to the macro?

Reply:
I'm now passing the local variables to the macro and this should now be working